index.js

```diff
@@ -84,6 +84,192 @@ app.post('/add/:quizName/:count', urlencodedParser, async (req, res) => {
     </html>`)    
 })
 
+////
+////
+////These are the /take routes
+
+app.post('/take', urlencodedParser, async(req, res) => {
+    console.log("This makes sure that this is running properly")  // TEMP
+    
+    var quizName = req.body.userName
```

Comment:
To start taking a quiz, the first page needs to ask for the quiz name and user name - two input boxes.

Reply:
That's correct, but `var quizName = req.body.userName` is wrong.

There must be two inputs coming from the static page, `req.body.userName` and `req.body.quizName`.

Reply:
I'll fix that. It seems like they're mixed up.